### DISCUSSION TITLE
the player does not work with the new version of QMPlay2  24.03.16

### DISCUSSION THREAD

**Discussion by goglm** [zaps166/QMPlay2#689] [General]
I launch a video file, the player appears for half a second and disappears. Tried both AppImage and AUR

In version 23.10.22 everything works fine.

Operating System: ArcoLinux 
KDE Plasma Version: 6.0.2
KDE Frameworks Version: 6.0.0
Qt Version: 6.6.2
Kernel Version: 6.6.22-1-lts (64-bit)
Graphics Platform: Wayland
Processors: 16 × AMD Ryzen 7 5800X3D 8-Core Processor
Memory: 31.2 ГиБ of RAM
Graphics Processor: AMD Radeon RX 6700 XT
Manufacturer: Gigabyte Technology Co., Ltd.
Product Name: B550M DS3H

**Comment by zaps166** [comment]
Try running using `gdb` and check the backtrace.

**Reply by goglm** [reply]
I can't do it

**Comment by goglm** [comment]
Could you add the old version to the AUR so that you don’t have to use AppImage?

**Comment by zaps166** [comment]
Without any help I can't fix the issue. Maybe try qmplay2-git from AUR first, it runs on Qt5. Also press F1 in software and check what's there (SIGSEGV)?

**Reply by goglm** [reply]
[20 Mar 2024 18:15:44.210] The Wayland connection experienced a fatal error

**Reply by zaps166** [reply]
Wayland again, use qmplay2-git from AUR.

**Comment by goglm** [comment]
I found a hint at this link https://github.com/zaps166/QMPlay2/issues/397. Installed openGL. Thank you I will know about F1. topic closed

**Reply by zaps166** [reply]
It should work, but every driver and Qt version are working differently on Wayland. This platform is very weird and immature. Try qmplay2-git, it still uses Qt5 and might be more stable.

**Reply by goglm** [reply]
Tried the same thing

**Reply by goglm** [reply]
Your player has very cool functionality. everything will get better with time. No more questions thank you very much

**Reply by zaps166** [reply]
Strange, I also checked on very similar configuration as yours on Wayland and Vulkan, no issues on my side. OpenGL is workaround only.
